packages/server/src/server.js

```diff
@@ -320,13 +320,11 @@ export default class Server {
   renderAndGetWindow (url, opts = {}, {
     loadingTimeout = 2000,
     loadedCallback = this.globals.loadedCallback,
-    ssr = this.options.render.ssr,
```

Comment:
Why?

Reply:
This option was not useful and leading to errors in tests: https://github.com/nuxt/nuxt.js/blob/dev/packages/server/src/jsdom.js#L53